### ISSUE TITLE
Amethyst will also pause when a non-tiled app is paused in debugger

### ISSUE THREAD

**Issue by interstateone** [ianyh/Amethyst#289]
If a non-tiled application is paused with a debugger and then I try to make Amethyst do something (anything, via keyboard shortcuts) it will hang until that paused application is resumed.

This seems specific enough to be different from #209, although maybe the solution ends up being the same.

This is an understandable behaviour, but since I'm often debugging applications on a computer running Amethyst, it quickly becomes frustrating when my keyboard shortcuts stop working.

Is it possible to have Amethyst ignore applications that aren't responding within a given timeout duration? I haven't looked into its implementation yet.

**Comment by ianyh** [comment]
Yeah, my off the cuff guess is that the accessibility frameworks just shut down when an app doesn't respond, so we can't actually send signals during that time. This happens when applications hang as well. It's possible that I'm doing something wrong and we can just treat an application that doesn't respond as effectively floating. Can look into it. I can understand how annoying that would be.
